app/mixins/event-wizard.js

```diff
@@ -59,59 +59,76 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
+        await data.copyright.save();
+      }
 
-    if (data.tax && data.tax.get('name')) {
-      let tax = this.setRelationship(data.tax, event);
-      if (event.get('isTaxEnabled')) {
-        await tax.save();
-      } else {
-        await tax.destroyRecord();
+      if (data.tax && data.tax.get('name')) {
+        let tax = this.setRelationship(data.tax, event);
+        if (event.get('isTaxEnabled')) {
+          await tax.save();
+        } else {
+          await tax.destroyRecord();
+        }
       }
-    }
 
-    if (data.stripeAuthorization && data.stripeAuthorization.get('stripeAuthCode')) {
-      let stripeAuthorization = this.setRelationship(data.stripeAuthorization, event);
-      if (event.get('canPayByStripe')) {
-        await stripeAuthorization.save();
-      } else {
-        await stripeAuthorization.destroyRecord();
+      if (data.stripeAuthorization && data.stripeAuthorization.get('stripeAuthCode')) {
+        let stripeAuthorization = this.setRelationship(data.stripeAuthorization, event);
+        if (event.get('canPayByStripe')) {
+          await stripeAuthorization.save();
+        } else {
+          await stripeAuthorization.destroyRecord();
+        }
       }
-    }
 
-    const bulkPromises = [];
+      const bulkPromises = [];
 
-    for (const property of ['tracks', 'sessionTypes', 'microlocations', 'customForms']) {
-      const items = data[property];
-      for (const item of items ? items.toArray() : []) {
-        bulkPromises.push(event.get('isSessionsSpeakersEnabled') ? item.save() : item.destroyRecord());
+      for (const property of ['tracks', 'sessionTypes', 'microlocations', 'customForms']) {
+        const items = data[property];
+        for (const item of items ? items.toArray() : []) {
+          bulkPromises.push(event.get('isSessionsSpeakersEnabled') ? item.save() : item.destroyRecord());
+        }
       }
-    }
 
-    for (const property of ['sponsors']) {
-      const items = data[property];
-      for (const item of items ? items.toArray() : []) {
-        bulkPromises.push(event.get('isSponsorsEnabled') ? item.save() : item.destroyRecord());
+      for (const property of ['sponsors']) {
+        const items = data[property];
+        for (const item of items ? items.toArray() : []) {
+          bulkPromises.push(event.get('isSponsorsEnabled') ? item.save() : item.destroyRecord());
+        }
       }
-    }
 
-    await Promise.all(bulkPromises);
+      await Promise.all(bulkPromises);
 
-    return event;
+      return event;
+    } else {
+      let errorObject = { 'errors': [] };
+      if (event.name === undefined || event.name === '') {
+        errorObject.errors.push({ 'detail': 'Event name has not been provided' });
+      }
+      if (event.locationName === undefined || event.locationName === '') {
+        errorObject.errors.push({ 'detail': 'Location has not been provided' });
+      }
+      if (event.startsAtDate === undefined || event.endsAtDate === undefined) {
```

Comment:
Can we have a check for ends >= starts ?

Reply:
@mrsaicharan1 That's a schema check and already handled on server, besides I am trying to keep this PR aligned with the FE issue